contrib/fwupd.spec.in

```diff
@@ -427,115 +427,9 @@ done
 %{_libdir}/fwupd-%{fwupdplugin_version}/libfwupdpluginbuiltin.so
```

Comment:
I think this should technically be renamed to `libfu_plugin_builtin.so` to keep with the style of naming used

Reply:
No can do; anything named `libfu_plugin_*.so` would be dlloaded by the plugin loader.